### PR TITLE
fixed some typos

### PR DESCRIPTION
**Pull request by SinithH**
fixed some typos
removed extra space between wrods

### CODE REVIEW

File: docs/advanced-latedelivery.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ When the text in the `Template` tab is in sync with the text in the `Test Contra
 
 ![Advanced-Late-3](assets/advanced/late3.png)
 
-Changes to the variables in the `Test Contract` are reflected in the Contract Data in realtime, and vice versa. For instance, change `Betty Buyer` to a different name in the contract text to see the `partyId` change in the contract data.
+Changes to the variables in the `Test Contract` are reflected in the Contract Data in real time, and vice versa. For instance, change `Betty Buyer` to a different name in the contract text to see the `partyId` change in the contract data.
```
Comment (irmerk) [inline docs/advanced-latedelivery.md]: This can be removed.

File: website/static/css/main.css [hunk 2]
```diff
@@ -782,6 +782,12 @@ blockquote {
     display: none !important;
   }
 }
+@media (max-width: 500px) {
+  img.logo {
+    content:url("../img/Accord-Project-logo-small.svg");
+  }
+}
+
```
Comment (irmerk) [inline website/static/css/main.css]: Can you provide context for why this is added?

File: website/static/img/Accord-Project-logo-small.svg [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 23.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Laag_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 136.1 135.8" style="enable-background:new 0 0 136.1 135.8;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#FFFFFF;}
+</style>
+<path class="st0" d="M88.1,135.8h48L92.3,0h-48l11.6,36.1H32.4L0,135.8h48l19.5-62.3h0.3L88.1,135.8z M85.8,9l38,117.8H94.7L56.7,9
+	H85.8z M65.1,64.5H47.3l-6.4-19.4h18L65.1,64.5z M41.5,126.8h-29l22.8-70.1l5.6,16.8h17.4L41.5,126.8z"/>
+</svg>
```
Comment (irmerk) [inline website/static/img/Accord-Project-logo-small.svg]: Can you provide context for why this is added?

File: website/static/css/main.css (hunk 2 above)
Comment (SinithH) [reply website/static/css/main.css]: ok removed it

File: website/static/img/Accord-Project-logo-small.svg (hunk 3 above)
Comment (SinithH) [reply website/static/img/Accord-Project-logo-small.svg]: ok removed it

**Comment by jeromesimeon** [comment]
No activity on this. Fixes have been folded into https://github.com/accordproject/techdocs/pull/198

**Pull request closed without merging**